examples/tutorials/forced_alignment_for_multilingual_data_tutorial.py

```diff
@@ -0,0 +1,444 @@
+)
+checkpoint = torch.load("model.pt", map_location="cpu")
+
+model.load_state_dict(checkpoint)
+model.eval()
+
+
+def get_emission(waveform):
+    # NOTE: this step is essential
+    waveform = torch.nn.functional.layer_norm(waveform, waveform.shape)
+
+    emissions, _ = model(waveform)
+    emissions = torch.log_softmax(emissions, dim=-1)
+    emission = emissions[0].cpu().detach()
+
+    # Append the extra dimension corresponding to the <star> token
+    extra_dim = torch.zeros(emissions.shape[0], emissions.shape[1], 1)
+    emissions = torch.cat((emissions.cpu(), extra_dim), 2)
+    emission = emissions[0].detach()
+    return emission, waveform
+
+
+# Construct the dictionary
+# '@' represents the OOV token, '*' represents the <star> token.
+# <pad> and </s> are fairseq's legacy tokens, which're not used.
+dictionary = {
+    "<blank>": 0,
+    "<pad>": 1,
+    "</s>": 2,
+    "@": 3,
+    "a": 4,
+    "i": 5,
+    "e": 6,
+    "n": 7,
+    "o": 8,
+    "u": 9,
+    "t": 10,
+    "s": 11,
+    "r": 12,
+    "m": 13,
+    "k": 14,
+    "l": 15,
+    "d": 16,
+    "g": 17,
+    "h": 18,
+    "y": 19,
+    "b": 20,
+    "p": 21,
+    "w": 22,
+    "c": 23,
+    "v": 24,
+    "j": 25,
+    "z": 26,
+    "f": 27,
+    "'": 28,
+    "q": 29,
+    "x": 30,
+    "*": 31,
+}
+
+
+######################################################################
+# Before aligning the speech with transcripts, we need to make sure
+# the transcripts are already romanized. The steps to romanize transcripts
+# using the "uroman" tool have been
+# listed below in comments.
+#
+
+# Here are the steps for downloading the uroman romanizer and using it to obtain normalized transcripts.
+# import re
+# def normalize_uroman(text):
+#     text = text.lower()
+#     text = text.replace("’", "'")
+#     text = re.sub("([^a-z' ])", " ", text)
+#     text = re.sub(' +', ' ', text)
+#     return text.strip()
+#
+# German example:
+# echo 'aber seit ich bei ihnen das brot hole brauch ich viel weniger schulze wandte sich ab die kinder taten ihm leid' > test.txt
+#
+# Chiense example:
+# echo '关 服务 高端 产品 仍 处于 供不应求 的 局面' > test.txt
+#
+# Polish example:
+# echo 'wtedy ujrzałem na jego brzuchu okrągłą czarną ranę dlaczego mi nie powiedziałeś szepnąłem ze łzami' > test.txt
+#
+# Portuguese example:
+# echo 'mas na imensa extensão onde se esconde o inconsciente imortal só me responde um bramido um queixume e nada mais' > test.txt
+#
+# Italian example:
+# echo "elle giacean per terra tutte quante fuor d'una ch'a seder si levò ratto ch'ella ci vide passarsi davante" > test.txt
+#
+# git clone https://github.com/isi-nlp/uroman
+# uroman/bin/uroman.pl < test.txt > test_romanized.txt
+#
+# file = "test_romanized.txt"
+# f = open(file, "r")
+# lines = f.readlines()
+# text_normalized = normalize_uroman(lines[0].strip())
+
+
+######################################################################
+# German example:
+# ~~~~~~~~~~~~~~~~
```

Comment:
For each language example, please print the original (un-romanized) sentence altogether. Then add the audio samples of segments. (make it more interactive)

Basically, to show what this API can do, include all the necessary information in one place.